types/evidence.go

```diff
@@ -288,12 +296,36 @@ func (e MockGoodEvidence) Equal(ev Evidence) bool {
 }
 func (e MockGoodEvidence) ValidateBasic() error { return nil }
 func (e MockGoodEvidence) String() string {
-	return fmt.Sprintf("GoodEvidence: %d/%s", e.EvidenceHeight, e.EvidenceAddress)
+	return fmt.Sprintf("GoodEvidence: %d/%s/%s", e.EvidenceHeight, e.Time(), e.EvidenceAddress)
 }
 
 // UNSTABLE
 type MockBadEvidence struct {
```

Comment:
should we merge mockbacdevidence and mockgoodevidence?

Reply:
why do u want to do it? should be fine, but just wonder what motivated this

Reply:
I don't see a need to have both with the difference being that one returns an error by default. If the evidence is meant to be bad then the time or block height can be set to a time that will make it error.
That was my thinking for it.